governance/SDK.md

```diff
@@ -10,7 +10,42 @@ The community is organized as follows:
 
 ## New maintainers
 
-TBD
+Maintainers are a crucial part of this community: they keep alive our projects
+and ensure both development and maintenance. We define a set of criteria,
+followed by a vote, to include an active contributor as maintainer of one or
+more sdk projects. The active contributor may propose himself to become a
+maintainer, although we encourage the community to include him as part of the
+group.
+
+The criteria for a contributor to `sdk-x` to become part of `sdk-x-maintainers`
+is:
+
+1. At least 20 reviews on eligible PRs
+1. At least 20 submitted eligible PRs as author
+
+When we refer to the term _eligible PR_, we explicit PRs with substantial
+contributions to the project. Because a _"substantial contribution"_ may vary
+between the different technologies used, each project defines in the
+`CONTRIBUTING.md` (available at the root of the Github Repository) what
+_eligible PR_ means for it.
```

Comment:
In general, I am not sure how comfortable I am with this. Documentation changes, adding test cases and other minor contributions are certainly valuable, and I would merit consistency, care and engagement, just as important as big sweeping pull requests.

Reply:
Sure, that's why I say "eligible PR" is something defined by each sdk. A typo in the doc for me it's not valuable, a bug fix with a new test case for me it's more than valuable. I don't say how this criteria should be defined here, because it's up to the single projects

Reply:
I see what you are saying. But this puts the onus on each SDK to define up front what constitutes grounds for an eligible PR, and personally, I don't feel comfortable doing that for sdk-javascript, given that I also think consideration should be given to other, "softer" contributions as I mention in my comment above.